tests/cluster/test.sh

```
@@ -129,7 +129,7 @@ for host in "${!hosts[@]}"; do
        ${XRDCP} ${LCLDATADIR}/${host}.ref ${hosts[$host]}/${RMTDATADIR}/${host}.ref
 
        for suffix in "${HTTP_SUFFIX[@]}"; do
-              ${CURL} -v -L ${hosts_http[$host]}/${RMTDATADIR}/${host}${suffix} -T ${LCLDATADIR}/${host}.ref
+              ${CURL} -v -L --retry 3 --retry-all-errors --retry-delay 1 ${hosts_http[$host]}/${RMTDATADIR}/${host}${suffix} -T ${LCLDATADIR}/${host}.ref
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)